tool/jt.rb

```diff
@@ -391,6 +391,7 @@ def help
           options                                    build the options
       jt build_stats [--json] <attribute>            prints attribute's value from build process (e.g., binary size)
       jt clean                                       clean
+      jt environment                                 prints the current environment
```

Comment:
`environment` is long and not so easy to type, I think we should abbreviate it to `jt env`

Reply:
Fixed.